README.md

```diff
@@ -131,3 +131,30 @@ We are using the dev environment to integrate all the approved code from master
 1. Test that everything still works on Test.
 
 Once production is live, we should plan to deploy to production at least once per sprint—if not more frequently.
+
+## Local build process for theme files
+
+There is a separate build process that is run in local development environments, to compile template files such as CSS and JS. The resulting files, in addition to the modified source files, are committed to the repo and deployed to upstream environments. They are always built locally, never as part of the main build process we've implemented with CircleCI.
+
+### Install dependencies 
+
+1. Install Node.js using the Mac or Windows installer available on nodejs.org.
+1. Open a terminal window (PowerShell on Windows) and run "npm install -g gulp-cli" to install gulp.js
```

Comment:
Also, you can use backtick (`) instead of double quotes so that commands are displayed with proper styling (here and in the following steps).